Doc/library/stdtypes.rst

```diff
@@ -1784,7 +1801,7 @@ expression support in the :mod:`re` module).
    Return the lowest index in the string where substring *sub* is found within
    the slice ``s[start:end]``::
 
-      >>> 'spam, spam, spam'.find('sp')
+      >>> 'spam, spam, spam'.find(',')
       0
```

Comment:
Is there something wrong here?

```python
>>> 'spam, spam, spam'.find(',')
4
>>> 'spam, spam, spam'.find('sp')
0
```

Reply:
Yes, these are more copy pasta errors. I will check this much more thoroughly the next time.